packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/components/_app-config/index.jsx.hbs

```diff
@@ -57,7 +57,9 @@ const AppConfig = ({children, locals = {}}) => {
             headers={headers}
             OCAPISessionsURL={`${appOrigin}/mobify/proxy/ocapi/s/${locals.site?.id}/dw/shop/v22_8/sessions`}
             {{#if answers.project.commerce.isSlasPrivate}}
-            clientSecret="_PLACEHOLDER_PROXY-PWA_KIT_SLAS_CLIENT_SECRET"
+            // Uncomment 'enablePWAKitPrivateClient' to use SLAS private client login flows.
+            // Make sure to also enable useSLASPrivateClient in ssr.js when enabling this setting.
+            enablePWAKitPrivateClient={true}
```

Comment:
this is enabled by default when a project is generated with private slas. Tte comment is telling the other way around when `enablePWAKitPrivateClient={true}` is commented out.

Reply:
Yes, in the templates used by `pwa-kit-create-app` we set `enablePWAKitPrivateClient` and don't comment it out.

The comment is there to keep the pwa-kit-create-app files more in sync with the one in `template-retail-react-app`.